notes/hub/create-pull-request.md

```diff
@@ -0,0 +1,13 @@
+## Create Pull Request
+
+Use the following command to create a pull request from a commit:
+
+> hub pull-request -p --no-edit -o
+
+... where the parameters have the following meaning:
+
+-p : push the latest commits
+
+--no-edit : use the first (chronologically) commit as the basis of pull request
+
+-o : open the pull-request in your browser
```

Comment:
think we should use some of the formatting features of markdown, for example making this a list and wrapping the flags in backticks to indicate code

```markdown
* `-p` : push the latest commits
* `--no-edit` : use the first (chronologically) commit as the basis of pull request
* `-o` : open the pull-request in your browser
```

Reply:
yes, I agree